_docs/en/tools/shell.md

```diff
@@ -0,0 +1,1462 @@
+- Parameter `-n|--max_count num`: Specifies the maximum number of values to retrieve.
+- Parameter `-t|--timeout_ms num`: Specifies the timeout duration for data retrieval.
+- Parameter `-d|--detailed`: Outputs detailed storage information for the data, including app_id, partition_index, and server_ip.
+- Parameter `-i|--no_value`: Retrieves only hash_key and sort_key without fetching the value.
+
+Examples:
+
+```
+>>> hash_scan pegasus cloud00 cloud01
+```
+
+### full_scan
+
+Perform a full scan of the table, with the option to specify filtering conditions for HashKey, SortKey, and Value. Results under the same HashKey will be sorted by SortKey. There is no guaranteed order between HashKeys.
+
+Usage:
+
+```
+USAGE: full_scan               [-h|--hash_key_filter_type anywhere|prefix|postfix]
+                               [-x|--hash_key_filter_pattern str]
+                               [-s|--sort_key_filter_type anywhere|prefix]
+                               [-y|--sort_key_filter_pattern str]
+                               [-v|--value_filter_type anywhere|prefix|postfix|exact
+                               [-z|--value_filter_pattern str]
+                               [-o|--output file_name]
+                               [-n|--max_count num]
+                               [-t|--timeout_ms num]
+                               [-d|--detailed]
+                               [-i|--no_value]
+```
+
+Explanation:
+
+- Please refer to the parameter explanations for more details [hash scan](#hash_scan).
+
+Examples:
+
+```
+>>> full_scan
+```
+
+### copy_data
+
+Insert the data from one table into another table one item at a time.
+
+Usage:
+
+```
+USAGE:  copy_data              <-c|--target_cluster_name str> <-a|--target_app_name str>
+                               [-p|--partition num] [-b|--max_batch_count num] [-t|--timeout_ms num]
+                               [-h|--hash_key_filter_type anywhere|prefix|postfix]
+                               [-x|--hash_key_filter_pattern str]
+                               [-s|--sort_key_filter_type anywhere|prefix|postfix|exact]
+                               [-y|--sort_key_filter_pattern str]
+                               [-v|--value_filter_type anywhere|prefix|postfix|exact]
+                               [-z|--value_filter_pattern str] [-m|--max_multi_set_concurrency]
+                               [-o|--scan_option_batch_size] [-n|--no_overwrite] [-i|--no_value]
+                               [-g|--geo_data] [-u|--use_multi_set]
+```
+
+Explanation:
+
+- The source table is specified using the `use` command, and the target table is executed using the `-c` and `-a` commands. The target table can be in another cluster. For detailed usage, refer to [Table Migration#copy_data_migration](/administration/table-migration#copy_data_migration). You can specify filtering conditions for HashKey, SortKey, and Value.
```

Comment:
Update the links, they are in lower case and joined by `-`, e.g. `/administration/table-migration#copy-data-migration`